Multisig/UI/App/MainTabBarViewController.swift

```diff
@@ -79,7 +79,12 @@ class MainTabBarViewController: UITabBarController {
 
         onFirstAppear(self)
 
-        WhatsNewHandler().whatsNewViewController?.present(on: self)
+        if (App.shared.intercomConfig.presentMessenger) {
+            App.shared.intercomConfig.presentMessenger = false
+            App.shared.intercomConfig.startChat()
+        } else {
+            WhatsNewHandler().whatsNewViewController?.present(on: self)
```

Comment:
@sche & @liliya-soroka Is it ok to not show the WhatsNew sceen if the user opened the app via push notification for Intercom chat? They will see it on the next app opening that is not done via PN.

Reply:
It is ok not to show

Reply:
I think it's even better not to show. Is it only for Intercome push notifications or for the tx push notifications too ?